### ISSUE TITLE
Add chrome.android.intentArgument option (only firefox.android.intentArgument is supported)

### ISSUE THREAD

**Issue by acreskeyMoz** [sitespeedio/browsertime#1328]
Browsertime supports optional android intent arguments for Firefox (`firefox.android.intentArgument`).

These are passed through `adb` and can be used to specify how the application will be launched.
For example, the browser can be launched to load a given website.

https://github.com/sitespeedio/browsertime/blob/0bfa3049293139e695d02a2ca0de594f3456d64c/lib/support/cli.js#L446

Example usage:
```
--firefox.android.intentArgument=-a
--firefox.android.intentArgument=android.intent.action.VIEW
--firefox.android.intentArgument=-d
--firefox.android.intentArgument="https://www.mozilla.org"
```

The bug is to extend this functionality so that Chrome on android is also supported

**Comment by sraturi** [comment]
I Don't have privileges to add myself as an assignee but I am working on this issue.

**Comment by soulgalore** [comment]
Thanks @sraturi

**Comment by acreskeyMoz** [comment]
I didn't realize that Browsertime uses Webdriver to launch the browsers on android.
I had thought that it sent `adb` commands and then attached.
(Of course this makes sense in retrospect).

So @sraturi is looking at ChromeOptions to add when launching Chrome.
But it may not fulfill our initial desire -- to launch Chrome as the Android OS does via intent.

**Comment by sraturi** [comment]
I am not sure if it makes any sense in continuing this ticket (at least scope wise, which is to add intent for chrome).

From what I gathered, when I try to even add `androidActivity` to the `web driver's experimentalOptions`, I get
`unknown error: known package com.android.chrome does not accept activity/process `

I also didn't find any for adding ADB commands to the web drivers. (though I'm still very new to selenium, and may have missed something).

I am still playing around with ChromeOptions but so far havent found anything solid.

Also tried the `--new-window` arg from https://peter.sh/experiments/chromium-command-line-switches/ but don't think it works on android.

I tried the homepage capability method from here https://stackoverflow.com/questions/19037535/how-can-you-start-a-browser-at-a-specific-url-using-the-selenium-webdriver  but no luck either


@acreskeyMoz any thoughts?
